### PR TITLE
log the scheduler version at the scheduler start

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java [hunk 1]
```diff
@@ -169,4 +176,17 @@ public static String getFileNameWithExtension(URL url) {
         return filename;
     }
 
+    public static Properties resolvePropertiesFile(String fileNamePath) {
+        Properties properties = new Properties();
+        try {
+            logger.debug("Loading properties from file " + fileNamePath);
+            FileInputStream stream = new FileInputStream(fileNamePath);
```
Comment (Mykhailenko) [inline common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java]: you can put FileInputStream stream = new FileInputStream(fileNamePath); to the try as it is AutoClosable, I guess..

File: common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java [hunk 2]
```diff
@@ -47,6 +52,8 @@
 @PublicAPI
 public class FileUtils {
 
+    private static Logger logger = Logger.getLogger(FileUtils.class);
```
Comment (Mykhailenko) [inline common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java]: you can add "final" as well

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java [hunk 3]
```diff
@@ -286,6 +295,12 @@ private static SchedulerAuthenticationInterface startScheduler(CommandLine comma
         return sai;
     }
 
+    private static String getSchedulerVersion() {
+        Properties properties = FileUtils.resolvePropertiesFile(GRADLE_PROPERTIES_PATH);
+        String version = properties.getProperty(VERSION_ATTRIBUTE_NAME);
```
Comment (Mykhailenko) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java]: you can put as a second argument default property like "not found".

File: common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java [hunk 4]
```diff
@@ -26,15 +26,20 @@
 package org.ow2.proactive.utils;
 
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.io.InputStream;
```
Comment (sonarqube-activeeon) [inline common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java]: MINOR Remove this unused import 'java.io.InputStream'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java [hunk 5]
```diff
@@ -26,15 +26,20 @@
 package org.ow2.proactive.utils;
 
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
```
Comment (sonarqube-activeeon) [inline common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java]: MINOR Remove this unused import 'java.io.FileNotFoundException'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java [hunk 6]
```diff
@@ -30,6 +30,7 @@
 import java.io.ByteArrayOutputStream;
 import java.io.File;
 import java.io.IOException;
+import java.io.InputStream;
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java]: MINOR Remove this unused import 'java.io.InputStream'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java [hunk 7]
```diff
@@ -81,8 +83,10 @@
 import org.ow2.proactive.scripting.ScriptResult;
 import org.ow2.proactive.scripting.SimpleScript;
 import org.ow2.proactive.utils.FileToBytesConverter;
+import org.ow2.proactive.utils.FileUtils;
 import org.ow2.proactive.utils.JettyStarter;
 import org.ow2.proactive.utils.PAMRRouterStarter;
+import org.ow2.proactive.utils.PAPropertiesLazyLoader;
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java]: MINOR Remove this unused import 'org.ow2.proactive.utils.PAPropertiesLazyLoader'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 4 issues
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by MaelAudren** [comment]
I applied all your suggestions Gleb. Thank you a lot for those improvements.

File: common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java [hunk 8]
```diff
@@ -47,6 +49,8 @@
 @PublicAPI
 public class FileUtils {
 
+    private final static Logger logger = Logger.getLogger(FileUtils.class);
```
Comment (sonarqube-activeeon) [inline common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java [hunk 9]
```diff
@@ -47,6 +49,8 @@
 @PublicAPI
 public class FileUtils {
 
+    private static final Logger logger = Logger.getLogger(FileUtils.class);
```
Comment (sonarqube-activeeon) [inline common/common-api/src/main/java/org/ow2/proactive/utils/FileUtils.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**